prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -154,6 +154,26 @@ periodics:
       args:
         - "-c"
         - "development/clusters-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false"
+- name: orphaned-vms-cleaner
+  #cron: "30 */4 * * *" # "At minute 30 past every 4th hour"
+  cron: "*/13 * * * *" # At every 13th minute. (Temporary in SAFE MODE)
+  labels:
+    preset-sa-gke-kyma-integration: "true"
+    preset-gc-project-env: "true"
+  decorate: true
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1
```

Comment:
Use same buildpack as for components: `eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd`